libs/design-system/src/navigation/NavMenu.cy.tsx

```diff
@@ -1,28 +1,27 @@
 import { NavMenu } from './NavMenu';
-import { TestWrapper } from '../../testing';
-import { ROUTES } from '../../constants/routes.enum';
```

Comment:
Should we export the ROUTES to `@novu/shared`?

Reply:
Right now I do not see the need it is only used in tests for the design system. But in the future, it might be needed.